src/libbson/src/bson/bson-clock.c

```diff
@@ -16,9 +16,6 @@
 
 
 #ifdef __APPLE__
-#include <mach/clock.h>
-#include <mach/mach.h>
-#include <mach/mach_time.h>
 #include <sys/time.h>
```

Comment:
Is this include still needed? It dates back to https://github.com/mongodb/libbson/commit/c6e981ef08f60bb505578fadfe96d328e4b2ea7a, although it's not clear if it was specific to the use of `mach_` functions since that commit also included it for the `defined(HAVE_CLOCK_GETTIME)` case.